config.json

```diff
@@ -7,7 +7,8 @@
   "general": {
     "useResourceIntegration": false,
     "toggleKey": "f1",
-    "toggleCommand": "phone"
+    "toggleCommand": "phone",
+    "defaultLanguage": "sv"
```

Comment:
This should probably be "en" by default.

Reply:
Yes :)

Reply:
Fixed